src/modules/endpoint/reducers/__tests__/morphContentLanguage.spec.js

```diff
@@ -291,15 +283,16 @@ describe('morphContentLanguage', () => {
   }): CityContentStateType => {
     let state = initialState
 
-    const setCityContentLocalizationAction: SetCityContentLocalizationType = {
-      type: 'SET_CITY_CONTENT_LOCALIZATION',
+    const initializeCityContentAction: InitializeCityContentActionType = {
+      type: 'INITIALIZE_CITY_CONTENT',
```

Comment:
Rename to SET so we stay more consistent with the naming of the other actions.

Reply:
Like `SET_CITY_CONTENT`? Imo it is not clear what it is then.

Reply:
@maxammann what do you propose?

Reply:
Hm yeah you are right, lets go with initialize